flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -2198,19 +2189,7 @@ var A320_Neo_UpperECAM;
             return "%";
         }
         getValue(_engine) {
-            const imbalance = SimVar.GetSimVarValue("L:A32NX_ENGINE_IMBALANCE", "number");
-            let n2Imbalance = 0;
-            if (parseInt(imbalance.toString().substr(0, 1)) == _engine) {
-                n2Imbalance = parseFloat(imbalance.toString().substr(5, 2)) / 100;
-            } else {
-                n2Imbalance = parseFloat(0);
-            }
-            const name = "ENG N2 RPM:" + _engine;
-            let percent = SimVar.GetSimVarValue(name, "percent") - n2Imbalance;
-            if (percent < 0.0) {
-                percent = 0;
-            }
-            return percent;
+			return Math.max(SimVar.GetSimVarValue("L:A32NX_ENGINE_N2:" + engineId, "percent"), 0);
```

Comment:
Looks like my code suggestion had an indentation error, sorry for that

Reply:
No problem. Just solved it!! :)